### PR TITLE
feat: root status in export patch log

### PR DESCRIPTION
**Pull request by validcube**
## Root status in export patch log

This add root status to the log.

### 📱 Testing 
* Samsung M23 5G (Android 13; One UI 5.1)

### 🔧 Close this or these!!
* #1405

### CODE REVIEW

File: lib/ui/views/installer/installer_viewmodel.dart [hunk 1]
```diff
@@ -191,19 +190,17 @@ class InstallerViewModel extends BaseViewModel {
       'Model: ${info['model']}',
       'Android version: ${info['androidVersion']}',
       'Supported architectures: ${info['supportedArch'].join(", ")}',
-
+      'Roo: ${isRooted ? 'Yes' : 'No'}',
```
Comment (validcube) [inline lib/ui/views/installer/installer_viewmodel.dart:193]:
```suggestion
      Root permissions: ${isRooted ? 'Yes' : 'No'}',
```

**Review by Ushie — CHANGES_REQUESTED** [review]
Please make sure not to let your formatter remove the empty lines from the log, they're there for readability

**Comment by validcube** [comment]
> Please make sure not to let your formatter remove the empty lines from the log, they're there for readability

Was going to do that once I'm available, thank for the heads up 👍

File: lib/ui/views/installer/installer_viewmodel.dart [hunk 2]
```diff
@@ -191,19 +190,17 @@ class InstallerViewModel extends BaseViewModel {
       'Model: ${info['model']}',
       'Android version: ${info['androidVersion']}',
       'Supported architectures: ${info['supportedArch'].join(", ")}',
-
+      Root permissions: ${isRooted ? 'Yes' : 'No'}',
```
Comment (validcube) [inline lib/ui/views/installer/installer_viewmodel.dart:193]:
```suggestion
      Root permissions: ${isRooted ? 'Yes' : 'No'}',
      
```

**Review by validcube — COMMENTED** [review]
Bleep

File: lib/ui/views/installer/installer_viewmodel.dart [hunk 3]
```diff
@@ -191,19 +190,20 @@ class InstallerViewModel extends BaseViewModel {
...
```
Comment (validcube) [inline lib/ui/views/installer/installer_viewmodel.dart:193]:
```suggestion
      'Root permissions: ${isRooted ? 'Yes' : 'No'}',
```

**Review by validcube — COMMENTED** [review]
Bloop

**Review by Ushie — APPROVED** [review]

**Pull request merged**